cg/meta/upload/gt.py

```diff
@@ -50,6 +50,6 @@ def _analysis_sex(self, hk_version: hk.models.Version) -> dict:
             data[sample_data['id']] = sample_data['predicted_sex']
         return data
 
-    def upload(self, data: dict):
+    def upload(self, data: dict, replace: bool=False):
```

Comment:
This is a function signature change. Did you check if this function is used anywhere else in `cg` to make sure it doesn't break anything else?

Reply:
I did, and it should be ok, I don't see it called from anywhere alse. But to be sure we should test it probably.